android/app/src/main/java/com/gpslogger/LocationModule.java

```diff
@@ -66,6 +66,7 @@ public String createLocationJSON(Location location) throws JSONException {
         JSONObject json = new JSONObject();
         json.put("longitude", location.getLongitude());
         json.put("latitude", location.getLatitude());
+	json.put("altitude", location.getAltitude());
```

Comment:
This is just in github messing with tabs and spaces, in my editor it's fine. I'll fix it, but are we using spaces instead of tabs and how many?